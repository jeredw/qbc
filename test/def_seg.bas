def seg = 42
def seg
