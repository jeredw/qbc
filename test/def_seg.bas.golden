(program 
  (statement 
    (def_seg_statement def seg = 
      (expr 
        (literal 42)))) \n 
  (statement 
    (def_seg_statement def seg)) \n)
